server/controllers/courses/deleteCourse.js

```diff
@@ -0,0 +1,23 @@
+const deleteCourse = require('../../database/queries/deleteCourse');
+
+exports.getStudent = (req, res) => {
+  const { id } = req.params;
+  deleteCourse(id)
+    .then((course) => {
+      if (!course.rowCount) {
+        res.status(404).send({
```

Comment:
@AbdallahAmmar96 Good use of 404 here and in the other endpoints, it's a very descriptive status code in the places that you're using it 👍